spec/factories/tasks.rb

```diff
@@ -1,5 +1,9 @@
 FactoryBot.define do
   factory :task do
-    
+    sequence(:title,'test_1')
+    content { 'テスト用に使用される内容です' }
+    status { 'todo' }
```

Comment:
enumの設定値だと分かるので、シンボルにしておいた方がベターです！
```suggestion
    status { :todo }
```

Reply:
655cf97 にて修正しました。